src/gql/queries/get-jira-custom-created-issues.graphql

```diff
@@ -0,0 +1,20 @@
+#import "../fragments/jiraTicket.graphql"
+
+query GetCustomCreatedIssues($taskId: String!, $execution: Int) {
+  task(taskId: $taskId, execution: $execution) {
+    annotation {
```

Comment:
task is missing an id and execution

Reply:
Apollo uses these fields to uniquely identify the returned task, So they always need to exist in a task response.